clients/drcachesim/common/options.cpp

```diff
@@ -333,3 +333,15 @@ droption_t<bool> op_reuse_verify_skip
  "Verifies every skip list-calculated reuse distance with a full list walk. "
  "This incurs significant additional overhead.  This option is only available "
  "in debug builds.");
+
+// XXX: replace function return address with function callstack
+droption_t<std::string> op_record_function
+(DROPTION_SCOPE_FRONTEND, "record_function", DROPTION_FLAG_ACCUMULATE, ";",
+ std::string("malloc|0|1") + ";" + std::string("tc_malloc|1|1") + ";" +
+ std::string("__libc_malloc|2|1"),
+ "Record invocation traces for the specified functions",
+ "Option value should fit this format"
```

Comment:
Repeating comment from before: please repeat the description in the long (please see the prior comment on the prior review)

Reply:
Done